spark-streaming-sql/query.sql

```diff
@@ -0,0 +1,59 @@
+WITH raw AS (
+  SELECT 
+    poc_id,
+    date,
+    beacon_report,
+    explode_outer(selected_witnesses) as witness,
+    true as selected
+    FROM lora_poc_v1
+  UNION ALL 
+  SELECT poc_id,
+    date,
+    beacon_report,
+    explode_outer(unselected_witnesses) as witness,
+    false as selected
+  FROM lora_poc_v1
+)
+SELECT
+    date,
+    selected,
+    base64(poc_id) as poc_id,
+    struct(
+      from_unixtime(beacon_report.received_timestamp / 1000) as received_timestamp_millis,
```

Comment:
As you're using `from_unixtime` here, I don't think the output is milliseconds anymore. It will be a string representation, right? So I'm worried that `_millis` in the output name isn't right.